tests/models/blip/test_modeling_blip.py

```diff
@@ -1111,6 +1111,7 @@ def test_inference_image_captioning(self):
             [30522, 1037, 3861, 1997, 1037, 2450, 3564, 2006, 1996, 3509, 2007, 2014, 3899, 102],
         )
 
+    @require_torch
```

Comment:
```suggestion
    @require_torch_gpu
```

Reply:
you might need to import it above from `testing_utils`

Reply:
Done